stdlib/Distributed/src/cluster.jl

```diff
@@ -990,20 +990,20 @@ function _rmprocs(pids, waitfor)
 end
 
 
-struct ProcessExitedException <: Exception end
-
 """
-    ProcessExitedException()
+    ProcessExitedException(pid::Int)
```

Comment:
let's call this `worker_id`, this sounds too much to me like it would be the `getpid` value